plugins/com.google.cloud.tools.eclipse.sdk.ui.test/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceAreaTest.java

```diff
@@ -155,6 +156,23 @@ public void testUi_sdkManagementFeature() {
     assertNotNull(sdkLocation);
     assertNotNull(sdkVersion);
   }
+  
+  @Test
+  public void testI2897() {
+    when(cloudSdkManager.isManagedSdkFeatureEnabled()).thenReturn(true);
```

Comment:
Add `when(preferences.getString(CloudSdkPreferences.CLOUD_SDK_MANAGEMENT)).thenReturn("MANUAL");`, and you don't need to manually call `chooseSdk.setSelection(true)` below. `sdkLocation.setText("")` is unnecessary too. I prefer this way, as it also verifies that the page is correctly initialized with the given preferences. Also, because you don't manually call `sdkLocation.setText("")`, you don't need `area.updateSelectedVersion()` either.